### ISSUE TITLE
Error: Couldn't find preset "stage-0"

### ISSUE THREAD

**Issue by ColeTownsend** [jongold/react-sketchapp-skpm-example#4]
Using Node 7.2.1, running `npm install` on a cleanly cloned repo, and then `npm run build` yields an error:
```
{ Error: Couldn't find preset "stage-0" relative to directory "/Users/ct/gits/react-sketchapp-skpm-example/node_modules/react-sketchapp"
    at error (/Users/ct/gits/react-sketchapp-skpm-example/node_modules/rollup/dist/rollup.js:170:12)
    at /Users/ct/gits/react-sketchapp-skpm-example/node_modules/rollup/dist/rollup.js:8937:6
  code: 'PLUGIN_ERROR',
  plugin: 'babel',
  id: '/Users/ct/gits/react-sketchapp-skpm-example/node_modules/react-sketchapp/lib/index.js' }
```

I've tried deleting the `node_modules` folder and running `npm cache clean`, then reinstalling to no effect.

**Comment by ghanbak** [comment]
s a m e

**Comment by mathieudutour** [comment]
try updating skpm to 0.7.3

**Comment by ColeTownsend** [comment]
Resolved by updating `skpm` to `0.7.3`

**Comment by jongold** [comment]
Sorry y'all - maybe made this repo public before I should have done; fixed now :)
